packages/common/src/util.js

```diff
@@ -108,6 +108,30 @@ util.ApiError = function(errorBody) {
 };
 
 /**
+ * Custom error type for partial errors returned from the API.
+ *
+ * @param {object} b - Error object.
+ */
+var PartialFailureError = createErrorClass('PartialFailureError', function(b) {
+  var errorObject = b;
+
+  this.errors = errorObject.errors;
+  this.response = errorObject.response;
+
+  var defaultErrorMessage = 'A failure occurred during this request.';
+  this.message = errorObject.message || defaultErrorMessage;
+});
+
+/**
+ * Wrap the PartialFailureError constructor so context isn't lost.
+ *
+ * @param {object} errorBody - Error object.
+ */
+util.PartialFailureError = function(errorBody) {
```

Comment:
Took this out. This is a bit scary, because I originally needed this wrapper when we implemented ApiError (this same pattern exists for that error type). The problem is, I don't remember why, except for my comment in the JSDocs, "...so that context isn't lost."

I haven't been able to produce any error when running our tests, so I took both wrapper functions `util.ApiError` and `util.PartialFailureError`, and just exposed what `create-error-class` returns naturally.

Can you think of any reason we needed the wrapper?